File: trunk/resources/scop_1.5.1/examples/receiver.cpp
// receiver.cpp - DMI - 7-9-02

#include <scop.h>

int main()
{
	int sock;
	char *msg;
	
	sock = scop_open("localhost", "receiver");
	while(1)
	{
		msg = scop_get_message(sock);
		printf("Received <%s>\n", msg);
		if(!strcmp(msg, "quit")) break;
		delete[] msg;
	}
	
	close(sock);
	return 0;
}

File: trunk/resources/scop_1.5.1/java/examples/Receiver.java
// Receiver.java - DMI - 20-9-02

import java.io.*;

public class Receiver
{
	public static void main(String[] args)
	{
		SCOP scop;
		String msg;

		scop = new SCOP("localhost", "Receiver");
		while(true)
		{
			msg = scop.get_message();
			System.out.println("Received <" + msg + ">");
			if(msg.equals("quit")) break;
		}

		scop.close();
	}
}